content/news/playtest-20190209.md

```diff
@@ -0,0 +1,34 @@
+
+A [new playtest build](/download/) is now available with a collection of fixes and improvements from [playtest-20190106](/news/playtest-20190106/).
```

Comment:
If I remember correctly, some people have been confused with the "from playtest-xyz" syntax before.  Does "fixes to and improvements over playtest-20190106" work?

Reply:
Done